script.js

```
@@ -1,88 +1,130 @@
+    if (taskText === '') return;
+
+    let taskList = document.getElementById('taskList');
+    let li = createTaskElement(taskText);
     
     taskList.appendChild(li);
     saveTask(taskText);
     taskInput.value = '';
-    taskInput.focus(); // Added focus back to input field after adding a task
+
+    return; // Unreachable code
+    console.log("This will never run");
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)